src/CarbonAware.Aggregators/src/CarbonAware/README.md

```diff
@@ -0,0 +1,58 @@
+# Carbon Aware Aggregator
+The Carbon Aware Aggregator serves as the aggregator layer between the controllers and the data sources. It's defined by the ICarbonAwareAggregator interface, and each function takes in either an IDictionary of properties or a CarbonAwareParameters object. The aggregator performs any necessary validation on the inputs. It then calls the necessary data source to get the data it needs and does an post-processing required before returning the result to the controller.
```

Comment:
Great work on the documentation here.  One of the key tasks we have is to continue to build out the documentation.  With other parties using the SDK, once they are up and running everything is fine, but until then there is a steep learning curve primarily due to documentation gaps, so every bit helps!

Reply:
@vaughanknight now that this is in, we could pull #136 out of draft. That PR added lots more documentation too to hopefully make it easier to new users